scripts/shared/clusters.sh

```diff
@@ -34,11 +33,10 @@ olm_version="${FLAGS_olm_version}"
 [[ "${FLAGS_prometheus}" = "${FLAGS_TRUE}" ]] && prometheus=true || prometheus=false
 [[ "${FLAGS_globalnet}" = "${FLAGS_TRUE}" ]] && globalnet=true || globalnet=false
 [[ "${FLAGS_registry_inmemory}" = "${FLAGS_TRUE}" ]] && registry_inmemory=true || registry_inmemory=false
-cluster_settings="${FLAGS_cluster_settings}"
 settings="${FLAGS_settings}"
 timeout="${FLAGS_timeout}"
 
-echo "Running with: k8s_version=${k8s_version}, olm_version=${olm_version}, olm=${olm}, globalnet=${globalnet}, prometheus=${prometheus}, registry_inmemory=${registry_inmemory}, cluster_settings=${cluster_settings}, settings=${settings}, timeout=${timeout}"
+echo "Running with: k8s_version=${k8s_version}, olm_version=${olm_version}, olm=${olm}, globalnet=${globalnet}, prometheus=${prometheus}, registry_inmemory=${registry_inmemory}, settings=${settings}, timeout=${timeout}"
```

Comment:
It would be good to include an example of the `friendlier YAML` version in the commit message so that any developers/users who run into issues can get pointers by looking at the commit message.

Please add an example of the new usage in the commit message.

Reply:
Please see usage documentation on the website - https://github.com/submariner-io/submariner-website/pull/609